arch/arm/src/common/arm_backtrace_unwind.c

```diff
@@ -728,11 +732,14 @@ int up_backtrace(struct tcb_s *tcb,
       frame.sp = frame.fp;
       frame.stack_top = (unsigned long)rtcb->stack_base_ptr +
```

Comment:
```suggestion
      frame.stack_base = (unsigned long)rtcb->stack_base_ptr;
      frame.stack_top = frame.stack_base + rtcb->adj_stack_size;
```

assign all stack base before top